scripts/release_ppa.sh

```diff
@@ -47,17 +51,34 @@ else
     branch=$1
 fi
 
+# Sanity check
+if ! grep ethereum /etc/dput.cf -q; then
+    echo "Error: Missing required entries in /etc/dput.cf (check top comment in $0 for more information)."
+    exit 1
+fi
+
 is_release() {
     [[ "${branch}" =~ ^v[0-9]+(\.[0-9]+)*$ ]]
 }
 
-keyid=379F4801D622CDCF
-email=builds@ethereum.org
+# source keyid and email from .release_ppa_auth
+if [ -e .release_ppa_auth ]
+then
+    source .release_ppa_auth
```

Comment:
Probably makes sense to add this to ``.gitignore`` then, doesn't it?

Reply:
good point